### DISCUSSION TITLE
How to monitor Mirth Connect? SNMP?

### DISCUSSION THREAD

**Discussion by alfonrodrisimon** [nextgenhealthcare/connect#5040] [Q&A]
Hi,

I would like to know how is it possible to monitor Mirth Connect for an external partner. Is somehow SNMP possible? It would be awesome that the infrastructure team receives some alerts if there is an error in any of the channels I created. 

Thanks ;)

**Comment by pacmano1** [comment]
SNMP is not built in.   You can roll your own solution.  You can license the Mirth Gold pack that has the advanced alerting plug in, or you can license https://consultzen.com/zen-insight/

**Reply by tonygermano** [reply]
Even the open source version has a basic Alerts section for catching errors. Alert destinations can be email addresses or channels. If you send to a channel, you can then forward it using any destination connector (http to a webhook, for example.)
